### ISSUE TITLE
FEAT: Statement execution logs

### ISSUE THREAD

**Issue by oojacoboo** [beekeeper-studio/beekeeper-studio#1085]
**Is your feature request related to a problem? Please describe.**
Currently, there isn't an easy way to see what statements are executed by the client.  For instance, just the other day there was a bug that resulted from the fact that BKS executes multiple update statements for changes to more than one column in a record.  There wasn't an easy way to see what was being executed, so I had to get logging output from the RDBMS.

**Describe the solution you'd like**
It'd be nice to be able to see what statements are being executed from changes in the UI.  One big benefit of this, is being able to copy the statement of execution from a change in the UI, then using that for migrations.  This is a very common workflow pattern, as it enables for you to make changes to your schema, and copy the statements for those changes into a migration file.

As it currently stands, you must re-create these statements manually, or rely on your logs from the RDBMS.  Standard/general query logs from an RDBMS can be quite chaotic and have a lot of output, so relying on it for this data isn't ideal.  Plus, setting this up and accessing it isn't exactly user-friendly.

In the current case, without these logs, you might be better off just manually writing your SQL and executing it, instead of using BKS' GUI for making schema changes.  That's unfortunate.

**Comment by rathboma** [comment]
Yes you are right, this would be great. It might take me a little while to get to it, but definitely should add this.
